.travis.yml

```diff
@@ -21,16 +21,18 @@ env:
   global:
     - PYENV_VERSION=3.6
     - PKG_TEST_PYTHON="--test-python=py36 --test-python=py27"
+    # TODO: deliberate that intake channel not used for releases (because packages will be on defaults?)
```

Comment:
Intake will be on defaults and the holoplot package should not depend on intake in any way. Only the docs and examples should.